### DISCUSSION TITLE
config/plugins/search/

### DISCUSSION THREAD

**Discussion by giscus** [pengzhanbo/vuepress-theme-plume#263] [docs-comment]
# config/plugins/search/

https://theme-plume.vuejs.press/config/plugins/search/

**Comment by sikefu-t** [comment]
我遇到个问题， 我的博客配置了 base前缀， 在搜索内容后 没法跳转到正确的url，请问这需要怎么进行配置啊

**Reply by pengzhanbo** [reply]
这是一个 bug，我先给这个 bug 创建一个 issue，下个版本修复。

**Reply by pengzhanbo** [reply]
已在新版本中修复

**Reply by MrHaoH** [reply]
大佬，我执行npm install vuepress-theme-plume@1.0.0-rc.108这个命令，报错：
npm error code ETARGET
npm error notarget No matching version found for vuepress-theme-plume@1.0.0-rc.108.
npm error notarget In most cases you or one of your dependencies are requesting
npm error notarget a package version that doesn't exist.

npm error A complete log of this run can be found in: C:\Users\Administrator\AppData\Local\npm-cache\_logs\2024-10-12T10_17_38_897Z-debug-0.log

没找到这个版本号，大佬，这个咋解决？

**Reply by pengzhanbo** [reply]
一般是由于 npm 镜像 还未拉取新的版本导致。要么等待 npm 镜像刷新，要么换回 npm 源。
后面我研究下怎么让 其它 npm 镜像在我发布后通知它刷新。

**Reply by sikefu-t** [reply]
大佬 厉害。 没问题了

**Comment by dingyuqi** [comment]
1. 请问搜索是否有可能显示更多的上下文信息?

    目前在搜索框中数据词组只能在下拉框中看到搜索到的标题. 而标题很多时候是重复的. 希望能看到更多的上下文信息来判断哪一个搜索结果是我需要的.
    
    例如此处出现了两个 "简介":
    ![image](https://github.com/user-attachments/assets/4a5d268d-f438-471b-b99f-49f886f0cfea)

2. 这个搜索是否包含标题以外的信息? 因为我看到带选项中都是文章标题.

**Reply by pengzhanbo** [reply]
本地搜索是通过 minisearch  这个库进行全文建立索引。如果引入更多的信息会导致生成的索引文件体积过大。如果需要使用到更精确的搜索结果，我建议使用 [Algolia DocSearch](https://theme-plume.vuejs.press/guide/features/content-search/#algolia-docsearch)
